iocBoot/templates/st.cmd.template

```diff
@@ -105,32 +106,41 @@ dbLoadRecords("EthercatMCdebug.template", "PREFIX=$(MOTOR_PREFIX), MOTOR_NAME=$(
 
 {% endfor %}
 {% endif %}
-{% if additional_db_files %}
+
+dbLoadRecords("iocSoft.db", "IOC={{prefix}}")
+dbLoadRecords("save_restoreStatus.db", "P={{prefix}}{{delim}}")
+
 cd "$(IOC_TOP)"
+
+{% if additional_db_files %}
 {% for db in additional_db_files %}
 dbLoadRecords("{{ db.file }}", "PORT={{ asyn_port }},PREFIX={{prefix}}{{delim}},IOCNAME=$(IOCNAME),{{ db.macros }}")
 
 {% endfor %}
 {% endif %}
-cd "$(IOC_TOP)"
-
-dbLoadRecords("db/iocSoft.db", "IOC={{prefix}}")
-dbLoadRecords("db/save_restoreStatus.db", "P={{prefix}}{{delim}}")
 
 # Setup autosave
 set_savefile_path( "$(IOC_DATA)/$(IOC)/autosave" )
 set_requestfile_path( "$(IOC_TOP)/autosave" )
+
 save_restoreSet_status_prefix( "{{prefix}}{{delim}}" )
 save_restoreSet_IncompleteSetsOk( 1 )
 save_restoreSet_DatedBackupFiles( 1 )
-set_pass0_restoreFile( "$(IOC).sav" )
-set_pass1_restoreFile( "$(IOC).sav" )
+set_pass0_restoreFile( "info_positions.sav" )
+set_pass0_restoreFile( "info_settings.sav" )
+set_pass1_restoreFile( "info_positions.sav" )
```

Comment:
`info_positions.sav` _probably_ shouldn't be restored in pass 1. This may cause the PLC to actually do something as restored values will be written into the PLC and not just reflected in EPICS as the default. 

Maybe opt-in? Not sure.

Reply:
I'm not sure if I understand these function calls. Does `set_pass1_restoreFile` set ALL the pvs in the `.sav` file, or only the ones marked as `pass1`? Or are the contents of these files determined by whether something is `pass1` or `pass0`?

Reply:
Take a look at the `.sav` files - they're just lists of PVs + values. As far as I know, if you say "restore this file at pass0/pass1", it'll restore everything in that file at the given pass.

Reply:
In that case, how do we differentiate between stuff that should be restored at pass1 and stuff that should be restored at pass0?

Reply:
Autosave docs say that pass0 becomes info_possitions and pass1 becomes info_settings, so we should just do each of these once in the correct pass, right?

Reply:
You got it!